src/Components/NavBar/NavBarLoggedInActions.tsx

```diff
@@ -159,6 +159,7 @@ export const NavBarLoggedInActionsQueryRenderer: React.FC<React.PropsWithChildre
     <FallbackErrorBoundary FallbackComponent={Placeholder}>
       <SystemQueryRenderer<NavBarLoggedInActionsQuery>
         environment={relayEnvironment}
+        placeholder={user ? <Placeholder /> : undefined}
```

Comment:
Fixed bug where nav flickers when checking whether logged in (after you've already logged in)